### PR TITLE
run rubocop auto correct

### PR DESCRIPTION
**Pull request by HParker**
@XanderStrike can rubocop autocorrect and did a few manual style changes. tests pass locally I am expecting CI to agree. branched from the upgrade-rails branch, so both can likely merge at once.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -14,25 +14,25 @@ def index
...
+      movie = Movie.find_by_id(id) || NilMovie.new('Unknown')
       [movie.title, val]
-    }
+    end
 
-    @views_by_day = View.where('created_at > ?', 1.week.ago).group_by { |u|
+    @views_by_day = View.where('created_at > ?', 1.week.ago).group_by do |u|
       u.created_at.localtime.beginning_of_day
-    }.reduce({}) { |h, (k,v)|
+    end.reduce({}) do |h, (k, v)|
```
Comment (HParker) [inline app/controllers/home_controller.rb]: ewww. I find this kinda gross.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,10 +1,9 @@
 module ApplicationHelper
-
-  def format_num n
-    n.to_s.reverse.gsub(/...(?=.)/,'\&,').reverse
+  def format_num(n)
+    n.to_s.reverse.gsub(/...(?=.)/, '\&,').reverse
   end
 
   def sub_url
-  	Setting.get(:url).content
```
Comment (HParker) [inline app/helpers/application_helper.rb]: so happy all the tabs are gone.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -14,65 +14,63 @@
...
+  #   config.mock_with :rspec do |mocks|
+  #     # Enable only the newer, non-monkey-patching expect syntax.
+  #     # For more details, see:
+  #     #   - http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
+  #     mocks.syntax = :expect
+  #
+  #     # Prevents you from mocking or stubbing a method that does not exist on
+  #     # a real object. This is generally recommended.
+  #     mocks.verify_partial_doubles = true
+  #   end
```
Comment (HParker) [inline spec/spec_helper.rb]: wow, I wasn't expecting this one. kinda neat I guess.

Comment (XanderStrike) [inline spec/spec_helper.rb]: Hmm not a fan of this, I'd prefer the comment block. I disabled comment indentation warnings on my rubocop.yml.

**Comment by XanderStrike** [comment]
:+1:

**Pull request merged**